Cargo.toml

```diff
@@ -16,7 +16,7 @@ members = [
 ]
 
 [workspace.dependencies]
-serde = { version = "1.0.193", features = ["derive"] }
+serde = { version = "1.0.214", features = ["derive"] }
```

Comment:
had to bump the serde version to fix a weird issue in the derive macro when `flatten` and `alias` were both used